castor/lib/core/src/main/scala/io/iohk/atala/castor/core/service/DIDService.scala

```diff
@@ -53,6 +53,7 @@ private class DIDServiceImpl(didOpValidator: DIDOperationValidator, nodeClient:
         .mapError(DIDOperationError.ValidationError.apply)
       operationOutput <- ZIO
         .fromFuture(_ => nodeClient.scheduleOperations(operationRequest))
+        .logError
```

Comment:
@patlo-iog, it would be nice to provide the information about the operation context.
As far as I know, it might be something like:
.logError("nodeClient.scheduleOperation")
What do you think?